release_notes_generator/builder.py

```diff
@@ -35,14 +35,14 @@ def __init__(self, records: dict[int, Record], changelog_url: str,
         :param formatter: A RecordFormatter object used to format the records.
         :param custom_chapters: A CustomChapters object representing the custom chapters.
         :param warnings: A boolean indicating whether to show warnings.
-        :param print_empty_chapters: A boolean indicating whether to print empty chapters.
         """
         self.records = records
         self.changelog_url = changelog_url
         self.formatter = formatter
         self.custom_chapters = custom_chapters
-        self.warnings = warnings
-        self.print_empty_chapters = print_empty_chapters
+
+        self.warnings = ActionInputs.get_warnings()
```

Comment:
In this section you combine two different docstrings:
You are missing a class documentation, where you should have all attr that are in the class.
You wrote a method doc, where you have now one extra `:param warnings` since you are mentioning only inputs.

Reply:
Doc string will be added/updated to final format in black, pylint issue.
We are discussing this topic in another PR.

Here, I have removed warnings from the method doc string as it is not received as a method parameter.